projects/storefrontlib/src/cms-components/misc/icon/icon.component.ts

```diff
@@ -49,6 +56,13 @@ export class IconComponent {
    */
   icon: SafeHtml;
 
+  /**
+   * The flip direction adds information to the DOM on whether it should flipped for a specific
+   * direction (ltr vs rtl). Typically, icons are ltr based, and only in case of rtl some of
+   * the icons will be flipped.
+   */
+  @HostBinding('attr.flip-at') flipDirection: DirectionMode;
```

Comment:
Subjective: The purpose of this binding is mainly styling. So more to me semantic would be binding CSS class instead of attribute.

Reply:
I can't argue with that. I preferred the previous ts implementation, but the final DOM is more important.